frontend-react/src/components/Admin/EditReceiverSettings.tsx

```diff
@@ -53,7 +107,10 @@ export function EditReceiverSettings({ match }: RouteComponentProps<Props>) {
                     break;
                 case "clone":
                     try {
-                        const data = JSON.stringify(orgReceiverSettings);
+                        // @ts-ignore
```

Comment:
NOTE: I could not for the life of me get this to stop complaining w/o the @ts-ignore, but it works as intended w/o any apparent side effects

Reply:
Object is possibly 'null'.
![image](https://user-images.githubusercontent.com/92405130/156608342-3b5dde7a-a0ee-4464-8cbe-51d36c9403eb.png)
Then slapping on the ?s gets me:
![image](https://user-images.githubusercontent.com/92405130/156608813-3c2dbea9-ab90-4fe2-a6c5-0d14e3e0977c.png)
@kevinhaube

Reply:
Your approach with the optional chain operator seems to not know the type of that ref. Try to cast the type in that `useRef()` otherwise it'll default to your initial value's type, in this case `never` for your initial value of `null`. If that doesn't work,

Reply:
@kevinhaube i did give this the college try, btw, and still could not get it to resolve ☹️